asreview/webapp/src/ProjectComponents/ReviewComponents/ReviewPageFinished.js

```diff
@@ -41,26 +34,69 @@ const StyledBox = styled(Box)(({ theme }) => ({
 
   [`& .${classes.text}`]: {
     textAlign: "center",
-    width: "40%",
+    width: "60%",
     [theme.breakpoints.down("md")]: {
       width: "80%",
     },
   },
 }));
 
 const ReviewPageFinished = (props) => {
+  const queryClient = useQueryClient();
+  const [recordEmpty, setRecordEmpty] = React.useState(false);
+
+  const handleClickExport = () => {
+    props.handleNavState("export");
+  };
+
+  const ifRecordPoolEmpty = React.useCallback(async () => {
+    const data = await queryClient.fetchQuery(
+      ["fetchRecord", { project_id: props.project_id }],
+      ProjectAPI.fetchRecord
+    );
+    setRecordEmpty(data["pool_empty"]);
+  }, [props.project_id, queryClient]);
+
+  React.useEffect(() => {
+    ifRecordPoolEmpty();
+  }, [ifRecordPoolEmpty]);
+
   return (
-    <StyledBox className={classes.root} aria-label="project review finished">
-      <img src={ElasFinished} alt="ElasFinished" className={classes.img} />
-      <Typography className={classes.textTitle} variant="h5">
-        Congratulations! You have finished this project.
-      </Typography>
-      <Typography className={classes.text}>
-        You have stopped reviewing and marked this project as finished. If you
-        want to resume the review, please <Link>update project status</Link>.
-      </Typography>
-    </StyledBox>
+    <Root aria-label="review page finished">
+      <Fade in>
+        <Stack
+          spacing={1}
+          sx={{
+            alignItems: "center",
+            height: "inherit",
+            justifyContent: "center",
+          }}
+        >
+          <img src={ElasFinished} alt="ElasFinished" className={classes.img} />
+          {!recordEmpty && (
+            <Stack spacing={1} sx={{ alignItems: "center" }}>
+              <Typography className={classes.textTitle} variant="h5">
+                Congratulations! You have finished this project.
+              </Typography>
+              <Typography className={classes.text}>
+                You have stopped reviewing and marked this project as finished.
+                If you want to resume the review, please{" "}
+                <Link>update project status</Link>.
+              </Typography>
+            </Stack>
+          )}
```

Comment:
@J535D165 When the project is marked as finished but still has unlabeled records, you will see something different (the link to update project status not working yet).

Reply:
Since marking a project as finished and finishing reviewing all records share the same property `reviewFinished`, when this property is true, the project review finished page needs to call `get_document` to decide which text should be shown.

Reply:
Ah sorry, I overlooked that. Nice solution.

Reply:
Suppose we add an additional property `allRecordsLabeled`, this would bring two things:
- Marking a project as finished only changes the value of `reviewFinished`.
- After reviewing all the records, the values of `allRecordsLabeled` and `reviewFinished` change.

After reviewing all the records, the project should not be marked as in review. This may call `get_document` again somewhere. Perhaps adding a property `allRecordsLabeled` to the project info looks to be a better solution?